bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/ApiBridgeHandler.java

```diff
@@ -219,7 +219,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
             throw new NetatmoException(String.format("%s: \"%s\"", e.getClass().getName(), e.getMessage()));
         } catch (TimeoutException | ExecutionException e) {
-            throw new NetatmoException(e, "Exception while calling %s", uri.toString());
+            if (retryCount > 0) {
+                logger.debug("Request timedout, retry counter : {}", retryCount);
+                return executeUri(uri, method, clazz, payload, retryCount - 1);
+            }
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                    "Request timedout - will attempt to reconnect later");
```

Comment:
Can you add this as i18n string and add missing space ("Request timed out")?

Reply:
Done